changelog/20.0/20.0.0/summary.md

```diff
@@ -4,6 +4,7 @@
 
 - **[Major Changes](#major-changes)**
   - **[Deletions](#deletions)** 
+    - [`--vreplication_tablet_type` flag is now deleted](#vreplication-tablet-type-deletion)
     - [`Pool Capacity Flags Deletion`](#pool-flags-deletion)
```

Comment:
Nit picking here, but I think it makes sense to not mention `deletion` and `deleted` again when the items are under the `Deletions` section